cli/tests/unit/event_test.ts

```diff
@@ -127,3 +132,30 @@ unitTest(function eventInspectOutput(): void {
     assertEquals(Deno.inspect(event), outputProvider(event));
   }
 });
+
+unitTest(function inspectEvent(): void {
+  // has a customInspect implementation that previously would throw on a getter
+  assertEquals(
+    Deno.inspect(Event.prototype),
+    `Event {
+  bubbles: [Getter/Setter],
```

Comment:
I'm not sure why these have empty setters defined... couldn't those be omitted?

Reply:
Yeah, those should be removed. Some legacy from when we understood how webidl works even less.

Reply:
Done in b2a8f21d848bfd598207bd991546f4a464173beb

Reply:
Nevermind. We need these because it causes an error to not have it when doing `e.target = something`, which doesn't error in chrome. I'm going to revert.

Reply:
Tested in the Chrome Dev Tools? That is non strict mode. It does error in Chrome too in ES modules or "use strict" scripts. That is how this snuck in in the first place :-)

Reply:
Oh, I didn’t think about that! Ok, yeah I’ll look into it on Thursday. There is a tests that assigns to target for some reason.

Reply:
@lucacasonato I was thinking about this. Maybe we should do it in a separate PR and minor release? It could possibly break some people's code. Thoughts?

Reply:
If you want, sure - but its a bug fix so I'd be happy to land it in a patch.

Reply:
Ok, lets merge this one and then I'll do a separate specific PR for that change so that it's not hidden under this one's title.